api/src/middleware/publications.js

```diff
@@ -53,6 +53,21 @@ const createPublicationValidation = [
       .if(body("yearPublished")
       .exists())
       .isLength(4),
+    body("category.type", "Error: Category type must not be empty.")
+      .notEmpty(),
+    body("category.type", "Error: Category type does not match any of [\"CONFERENCE\", \"JOURNAL\"].")
```

Comment:
The \" parts show up in the error message, so maybe it's better to remove them instead or use single quotes only like ['CONFERENCE', 'JOURNAL']